app/email_alert_api_docs.rb

```diff
@@ -0,0 +1,32 @@
+class EmailAlertApiDocs
+  PAGES = {
+    "analytics" => "Analytics",
+    "api" => "Available endpoints",
+    "fields" => "Field Reference",
+    "matching-content-to-subscriber-lists" => "Matching content to subscriber lists",
+    "tasks" => "Tasks",
+    "transition-from-govdelivery-to-notify" => "Transition from Govdelivery to Notify",
+    "troubleshooting" => "Troubleshooting",
+  }.freeze
+
+  def self.pages
+    PAGES.map { |k, v| Page.new(k, v) }
```

Comment:
```suggestion
    PAGES.map { |filename, title| Page.new(filename, title) }
```
Not a blocker but it makes this clearer and reflects the `Page` attributes.